cli/bin/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set +x
+
+BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "${BIN_DIR}/.."
+
+if [ -z "$GOPATH" -o -z "$(which go)" ]; then
```

Comment:
This part is also found in build-go.sh and it is repeated here. I don't like bash include files but maybe we should add it in one. Let's verify that this works for the jenkins stuff.

Reply:
I would almost prefer that we don't add it to `build-go.sh` so that you can build the CLI without invoking the tests (or commenting out lines then forgetting to uncomment them). FWIW I usually build the CLI then just copy it to my `.dcos/` dir for testing the CLI.